CRM/Report/Form/Contribute/SoftCredit.php

```diff
@@ -192,6 +192,72 @@ public function __construct() {
         ],
         'grouping' => 'contact-fields',
       ],
+      'civicrm_address' => [
+        'dao' => 'CRM_Core_DAO_Address',
+        'fields' => [
+          'address_creditor' => [
+            'title' => ts('Soft Credit Street Address'),
+            'name' => 'street_address',
+            'alias' => 'acredit',
+            'default' => TRUE,
+          ],
+          'city_creditor' => [
+            'title' => ts('Soft Credit City'),
+            'name' => 'city',
+            'alias' => 'acredit',
+            'default' => TRUE,
+          ],
+          'postal_code_creditor' => [
+            'title' => ts('Soft Credit Postal Code'),
+            'name' => 'postal_code',
+            'alias' => 'acredit',
+            'default' => TRUE,
+          ],
+          'country_creditor' => [
+            'title' => ts('Soft Credit Country'),
+            'name' => 'country_id',
+            'alias' => 'acredit',
+            'default' => TRUE,
+          ],
+          'state_creditor' => [
+            'title' => ts('Soft Credit State'),
+            'name' => 'state_province_id',
+            'alias' => 'acredit',
+            'default' => TRUE,
+          ],
+          'address_constituent' => [
+            'title' => ts('Contributor\'s Street Address'),
+            'name' => 'street_address',
+            'alias' => 'aconst',
+            'no_repeat' => TRUE,
+          ],
+          'city_constituent' => [
+            'title' => ts('Contributor\'s City'),
+            'name' => 'city',
+            'alias' => 'acredit',
```

Comment:
It's a little hard to see what the test fail is because the log truncates the error, but I'd guess it's because these aliases and the ones below are the wrong alias (should be aconst).